Loop/Settings/GeneralSettingsView.swift

```diff
@@ -95,14 +95,14 @@ struct GeneralSettingsView: View {
             }
 
             Section {
-                Picker("Animation Speed", selection: $animationConfiguration) {
+                Picker("Animation speed", selection: $animationConfiguration) {
                     ForEach(AnimationConfiguration.allCases) { configuration in
                         Text(configuration.name)
                     }
                 }
             }
 
-            Section("Loop's icon") {
+            Section("\(Bundle.main.appName)'s icon") {
```

Comment:
Actually, since the user already knows that this is Loop, can you change "Loop's Icon" to simply "Icon"?

Reply:
Updated to "App Icon".